sdk/textanalytics/ai-text-analytics/samples/typescript/src/beginAnalyzeHealthcare.ts

```diff
@@ -42,18 +42,16 @@ export async function main() {
       for (const entity of result.entities) {
         console.log(
           `\t- Entity ${entity.text} of type ${entity.category} ${
-            entity.relatedHealthcareEntities.size > 0
-              ? "and it is related to the following entitites"
-              : ""
+            entity.relatedEntities.size > 0 ? "and it is related to the following entities" : ""
           }`
         );
-        for (const edge of entity.relatedHealthcareEntities) {
+        for (const edge of entity.relatedEntities) {
           console.log(`\t\t- ${edge[0].text} with the relationship being ${edge[1]}`);
         }
-        if (entity.dataSource && entity.dataSource.length > 0) {
+        if (entity.dataSources.length > 0) {
```

Comment:
I saw there is no more null check for data source. Just curious if it is on purpose.

Reply:
I made `dataSources` a required field so there is no longer a need to check if it exists.